doc/reducing-memory-usage.md

```diff
@@ -0,0 +1,8 @@
+In-memory caches
+----------------
+
+The size of some in-memory caches can be reduced. As caches trade off memory usage for performance, usually reducing these have a negative effect on performance.
+
+- `-dbcache=<n>` - the UTXO database cache size, this defaults to `450` (`100` before 1.0.15). The unit is MiB (1024)
```

Comment:
The `(1024)` seems like a typo. Maybe expand to `(where 1 GiB = 1024 MiB)` or something similar?

Reply:
Done